pillar/apps/redash.sls

```diff
@@ -111,12 +113,7 @@ uwsgi:
         - env: '%(_)'
         - endfor: ''
         - attach-daemon2: >-
-            cmd=/usr/local/pyenv/versions/{{ python_version }}/bin/celery worker -A redash.worker -B -c2 -Qqueries,celery --pidfile /opt/{{ app_name }}/celery.pid -Ofair,
```

Comment:
Why are we getting rid of this?

Reply:
Because it's unnecessary. I'm just combining all of the queues into the one command.